src/components/TextPill.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import Text from './Text';
+import styles from '../styles/styles';
+import stylePropTypes from '../styles/stylePropTypes';
+
+const propTypes = {
+    /** Text to show */
+    children: PropTypes.string,
```

Comment:
Can we rename this to something related to text say `title`? `text`. etc.?

`children` is generally used for nodes or React components and looks misleading here.

Reply:
Done.